css/styles.css

```diff
@@ -12,3 +12,8 @@
     padding-bottom: 0px;
     margin-bottom: 0px;
 }
+
+.gm-style-iw {
+  width: 324px !important;
```

Comment:
use !important only when you really mean it. Here you can do it better ;)

Reply:
right, adding more details in the infowindow would require that to be removed :)